include/zephyr/net/ocpp.h

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) 2024 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file ocpp.h
+ *
+ * @defgroup ocpp_api OCPP library
+ * @ingroup networking
+ * @{
+ * @brief OCPP Charge Point Implementation
+ *
+ * @note The implementation assumes Websocket module is enabled.
+ *
+ * @note By default the implementation uses OCPP version 1.6.
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_OCPP_H
+#define ZEPHYR_INCLUDE_NET_OCPP_H
```

Comment:
All the header files should follow similar naming for guard defines so this should be
```
#ifndef ZEPHYR_INCLUDE_NET_OCPP_H_
```
so you should add `_` at the end. Not sure if there is some tool that checks these.